analysis/analysisrunner.cpp

```diff
@@ -227,11 +240,11 @@ QString AnalysisRunner::humanFileSize() const
 {
     int unit;
     const char *units [] = {" Bytes", " kB", " MB", " GB"};
-    quint64 size = fileSize(); // or whatever
+    qint64 size = fileSize(); // or whatever
 
     for (unit=-1; (++unit<3) && (size>1023); size/=1024);
 
-    return QString::number(size, 'f', 1) + QString(units[unit]);
+    return QString::number(size, 'f', 1) + QString::fromLatin1(units[unit]);
```

Comment:
QString(const char *str) constructor creates QString from UTF-8,

compared to

QString::fromLatin1(const char *str, int size = -1) that creates QString from Latin-1 charset.

http://doc.qt.io/qt-5/qstring.html#QString-7
http://doc.qt.io/qt-5/qstring.html#fromLatin1